docs/pages/deploy-a-cluster/helm-deployments/gcp.mdx

```diff
@@ -185,11 +185,16 @@ Go back to the "Service Accounts" view in Google Cloud IAM & Admin.
 
 Find the path where the JSON private key was just saved (likely your browser's default "Downloads" directory).
 
-Use `kubectl` to create the secret, using the path to the JSON private key:
+Use `kubectl` to create the `teleport` namespace, set its security policy and
+create the secret, using the path to the JSON private key:
```

Comment:
The comma placement here makes it hard to know (without prereq knowledge on the subject) which clauses are related. Ignore my suggestion if ignorance is my folly.

```suggestion
Use `kubectl` to create the `teleport` namespace, set its security policy, and
create the secret using the path to the JSON private key:
```